### PR TITLE
refactor: migrate random sparse encoder[149]

### PR DESCRIPTION
**Pull request by theUnkownName**
Migrate RandomSparseEncoder

### CODE REVIEW

**Comment by JoanFM** [comment]
Check this PR https://github.com/jina-ai/jina-hub/pull/148, since `TransformEncoder` is going to be moved back to `core`

File: encoders/numeric/RandomSparseEncoder/manifest.yml [hunk 1]
```diff
@@ -8,5 +8,5 @@ vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
 version: 0.0.1
 license: apache-2.0
-keywords: [encoder]
+keywords: [ -numeric -sklearn]
```
Comment (JoanFM) [inline encoders/numeric/RandomSparseEncoder/manifest.yml:11]: these keywords should be comma separated values

**Comment by JoanFM** [comment]
Your commit did not pass the commit lint check, please make it follow the guidelines.


Bad commit message, please refer to:
- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
#553 :bookmark: Commit Message Guideline for the First Time Contributor
jina-ai/jina | Jun 21st | Added by GitHub
CONTRIBUTING.md
jina-ai/jina | Added by GitHub

File: encoders/numeric/RandomSparseEncoder/manifest.yml [hunk 2]
```diff
@@ -8,5 +8,5 @@ vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
 version: 0.0.1
 license: apache-2.0
-keywords: [ -numeric -sklearn]
+keywords: [ -numeric, -sklearn]
```
Comment (JoanFM) [inline encoders/numeric/RandomSparseEncoder/manifest.yml:11]: and not `-`

**Review by hanxiao — APPROVED** [review]

**Pull request merged**